connection.go

```diff
@@ -258,6 +267,10 @@ func (mc *mysqlConn) interpolateParams(query string, args []driver.Value) (strin
 }
 
 func (mc *mysqlConn) Exec(query string, args []driver.Value) (driver.Result, error) {
+	return mc.ExecContext(backgroundCtx(), query, args)
+}
+
+func (mc *mysqlConn) ExecContext(ctx mysqlContext, query string, args []driver.Value) (driver.Result, error) {
```

Comment:
Here is an example of what I mean: https://github.com/go-sql-driver/mysql/commit/e1cf7dbff43726972186c542c2a9a85c13684bd7